setup.py

```diff
@@ -506,7 +506,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     "facebook": facebook,
     "ftp": [],
     "google": google,
-    "grpc": grpc,
+    "grpc": grpc + google,
```

Comment:
? Why do we need the entire google cloud suite for gprc hooks? That seems excessively heavy.

Reply:
Because this hook uses google https://github.com/apache/airflow/blob/master/airflow/providers/grpc/hooks/grpc.py

```
from google import auth as google_auth
from google.auth import jwt as google_auth_jwt
from google.auth.transport import (
    grpc as google_auth_transport_grpc, requests as google_auth_transport_requests,
)
```

Surely we can add just one ore two libraries instead to grpc - google-auth related ones.

Possibly those two will be enough? WDYT?

```
'google-auth>=1.0.0, <2.0.0dev',
'google-auth-httplib2>=0.0.1',
```

Reply:
I push it in a moment.